project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.4.9
+sbt.version=1.6.2
```

Comment:
The Catalogue states latest version is 1.5.8. Are we happy going up to 1.6.2?

Reply:
I referred to one of the PR's, happy to change it to 1.5.8. Thanks

Reply:
We upgraded most of project to 1.6.2 not sure what is wrong in catalogue and it is saying that latest is 1.5.8

Reply:
It seems quite a few services are 1.6.2 as shown [here](https://catalogue.tax.service.gov.uk/dependencyexplorer/results?group=org.scala-sbt&artefact=sbt&team=&flag=latest&scope=build&versionRange=%5B0.0.0%2C1.7.1%5D&asCsv=false) but probably Jenkins has 1.5.8 installed or something which might be why the catalogue hasn't been updated.

Reply:
Looks safe to goto 1.6.2 https://catalogue.tax.service.gov.uk/dependencyexplorer/results?group=org.scala-sbt&artefact=sbt&team=&flag=latest&scope=build&versionRange=%5B0.0.0%2C1.7.1%5D&asCsv=false

Reply:
I changed it to 1.5.8, should i upgrade it back to 1.6.2?

Reply:
yes please!